frontend/ui/bidi.lua

```diff
@@ -283,4 +284,18 @@ function Bidi._filepath_rtl(path)
     return Bidi.ltr(Bidi._path(dirpath) .. Bidi._filename_auto_ext_right(filename))
 end
 
+function Bidi.invert()
+    if G_reader_settings:isTrue("invert_ui_layout_mirroring") then
+	    Bidi._mirrored_ui_layout = not Bidi._mirrored_ui_layout
+	    Bidi._invert = true
+    end
+end
+
+function Bidi.resetinvert()
+    if Bidi._invert then
```

Comment:
The flag better be named `_inverted` so it reads like a state and not an action.
I wonder if there isn't a better english word for this action. "invert" to me feels to much like "mirror". I know we're inverted the mirroring, but may be swap or toggle would make it clearer ?